services/db/migrations/019_cvm.up.sql

```diff
@@ -0,0 +1,29 @@
+create table `cvm_block`
+(
+    block decimal(65) not null primary key,
+    canonical_serialization varbinary(64000) not null,
+    created_at              timestamp        not null default current_timestamp
+);
+
+create table `cvm_transactions`
+(
+    id             varchar(50)     not null primary key,
+    blockchain_id  varchar(50)     not null,
+    created_at     timestamp       not null default current_timestamp
+);
+
+create table `cvm_address`
```

Comment:
Nit: table names should be plural. (Or if we wanted to go with singular table names, we should decide that and plan to change the others at some point in the future).